src/utils/configValidators/eitherNilOrNonEmpty.js

```diff
@@ -13,7 +13,7 @@ governing permissions and limitations under the License.
 import isNil from "../isNil";
 import isNonEmptyString from "../isNonEmptyString";
 
-export default (config, key, currentValue) => {
+export default (key, currentValue) => {
   if (isNil(currentValue)) {
```

Comment:
I think this should be `if (currentValue === undefined){`. I think validation should fail if the user passes in `null`.

Reply:
Oh wait.....nevermind on this one.